src/main/java/de/dennisguse/opentracks/services/TrackRecordingManager.java

```diff
@@ -161,7 +161,7 @@ boolean onNewTrackPoint(TrackPoint trackPoint) {
                 return true;
             }
 
-            if (trackPoint.hasSensorData() || (distanceToLastTrackLocation.greaterOrEqualThan(recordingDistanceInterval) && trackPoint.isMoving())) {
+            if (distanceToLastTrackLocation.greaterOrEqualThan(recordingDistanceInterval) && trackPoint.isMoving()) {
```

Comment:
Could it result in a low heart rate or cadence precision? Shouldn't we compute the average or something like this between last point recorded and this one?

Reply:
You are totally right - we are should do some averaging there I guess.
We were never doing it correctly, as we always stored the most recent reported sensor value when a GPS location was received.
The rest is dropped.

I created a new issue for this: #1029.

@rgmf Would it be okay for you to ignore this for now?

Reply:
It would be okay for me. In fact, separate this, sounds like a good idea.